### PR TITLE
8015886: java/awt/Focus/DeiconifiedFrameLoosesFocus/DeiconifiedFrameLoosesFocus.java sometimes failed on ubuntu

### PR DESCRIPTION
**Pull request by azvegint**
This test can fail pretty consistently on some slow systems. Increased delay fixes the issue. 
We do have similar fix in #260

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8015886](https://bugs.openjdk.java.net/browse/JDK-8015886): java/awt/Focus/DeiconifiedFrameLoosesFocus/DeiconifiedFrameLoosesFocus.java sometimes failed on ubuntu

### Reviewers
 * [Alexander Zuev](https://openjdk.java.net/census#kizune) (@azuev-java - **Reviewer**) ⚠️ Review applies to 10ca677e027f3bfd2853240742b57c0774805eb6
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk17 pull/272/head:pull/272` \
`$ git checkout pull/272`

Update a local copy of the PR: \
`$ git checkout pull/272` \
`$ git pull https://git.openjdk.java.net/jdk17 pull/272/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 272`

View PR using the GUI difftool: \
`$ git pr show -t 272`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk17/pull/272.diff">https://git.openjdk.java.net/jdk17/pull/272.diff</a>

</details>

### CODE REVIEW

**Comment by azuev-java** [comment]
I am not sure if we supposed to bump up the copyright year or it is supposed to be done with some script later, but i usually do it myself just to be sure.

**Comment by azvegint** [comment]
> I am not sure if we supposed to bump up the copyright year or it is supposed to be done with some script later, but i usually do it myself just to be sure.

updated

**Review by azuev-java — APPROVED** [review]

File: test/jdk/java/awt/Focus/DeiconifiedFrameLoosesFocus/DeiconifiedFrameLoosesFocus.java [hunk 1]
```diff
@@ -79,10 +79,12 @@ void test() {
         frame.setExtendedState(Frame.ICONIFIED);
 
         Util.waitForIdle(robot);
+        robot.delay(200);
```
Comment (mrserb) [inline test/jdk/java/awt/Focus/DeiconifiedFrameLoosesFocus/DeiconifiedFrameLoosesFocus.java:82]: I remember some similar issues, so I 'grep" our tests. Looks like usually a bigger delay is used, like 500 or even 1 second.

Comment (prsadhuk) [reply test/jdk/java/awt/Focus/DeiconifiedFrameLoosesFocus/DeiconifiedFrameLoosesFocus.java:82]: Normally we use delay of 1 sec after frame.setvisible and 500ms after events. 

Can you also please rectify the wildcard imports? Did you try to give CI job running this test for several iterations in all platforms, if yes, can you post it in JBS?

**Review by prsadhuk — COMMENTED** [review]
Looks good. BTW, normally in swing we also do dispose of the frame after test is over in finally block...Not sure in awt Frame...It may not be required if you are running standalone but in a group of CI run, it might prove effective..

**Comment by azvegint** [comment]
> Looks good. BTW, normally in swing we also do dispose of the frame after test is over in finally block...Not sure in awt Frame...It may not be required if you are running standalone but in a group of CI run, it might prove effective..

looks like jtreg handles this, but this might be helpful specially for standalone run.

**Comment by azvegint** [comment]
CI job run is green.

**Review by mrserb — APPROVED** [review]

**Pull request closed without merging**